src/v2/Apps/Order/Components/BuyerGuarantee.tsx

```diff
@@ -0,0 +1,23 @@
+import { CircleBlackCheckIcon, Flex, Link, Text, color } from "@artsy/palette"
+import React from "react"
+
+export const BuyerGuarantee: React.FC = () => {
+  return (
+    <Flex p={2} my={1} backgroundColor={color("black10")}>
+      <CircleBlackCheckIcon mr={1} />
+      <Flex flexDirection="column">
+        <Text variant="mediumText">Your purchase is protected.</Text>
+        <Text variant="text" color="black60">
+          Learn more about{" "}
+          <Link
+            target="_blank"
+            rel="noopener noreferrer"
+            href="https://support.artsy.net/hc/en-us/articles/360048946973-How-does-Artsy-protect-me"
```

Comment:
Maybe we can make this link a constant and move it to either `BuyerGuarantee` component.

Reply:
in https://github.com/artsy/force/pull/6675/commits/f2d031d1a4fc8648fafe2e5b618f8c1ec7fc8055